### PR TITLE
Updated Images and overview page - addressed the 3 minor things

### PR DESCRIPTION
**Pull request by StevenMih**
Updating per @aweisberg 

image

### CODE REVIEW

**Comment by StevenMih** [comment]
@aweisberg @rschlussel 
please review thanks

File: website/static/overview.html [hunk 1]
```diff
@@ -47,77 +47,61 @@ <h1>Overview</h1>
...
+            Presto is a distributed system that runs on a cluster of machines. It enables 
+	analytics on large amounts of data. With Presto, access and query data in place on
+	many data different data sources (see image below). Presto supports ANSI SQL.
+        </p>
+
+   <!--suppress CheckImageSize -- larger file for retina -->
+        <img src="static/presto-overview-2.png" width="80%" alt="Presto System Overview">
+
+	<p>
+         A full installation includes a coordinator and multiple workers.
```
Comment (aweisberg) [inline website/static/overview.html:62]: This still says installation?

File: website/static/overview.html [hunk 2]
```diff
@@ -47,77 +47,61 @@ <h1>Overview</h1>
...
     <div class="">
-        <h1>Introduction</h1>
+        <h1>Introduction to Presto</h1>
 
         <p>
-            Presto is a distributed system that runs on a cluster of machines.
-            A full installation includes a coordinator and multiple workers.
+            Presto is a distributed system that runs on a cluster of machines. It enables 
+	analytics on large amounts of data. With Presto, access and query data in place on
+	many data different data sources (see image below). Presto supports ANSI SQL.
```
Comment (aweisberg) [inline website/static/overview.html:55]: This still doesn't have the alternative wording https://github.com/prestodb/prestodb.github.io/pull/99#discussion_r465395718

File: website/static/overview.html [hunk 3]
```diff
@@ -47,77 +47,61 @@ <h1>Overview</h1>
...
             Queries are submitted from a client such as the Presto CLI to the
             coordinator. The coordinator parses, analyzes and plans the query
             execution, then distributes the processing to the workers.
         </p>
 
         <!--suppress CheckImageSize -- larger file for retina -->
-        <img src="static/presto-overview.png" width="80%" alt="Presto Installation Overview">
-
-        <h1>Requirements</h1>
+        <img src="static/presto-architecture.png" width="80%" alt="Presto Installation Overview">
```
Comment (aweisberg) [inline website/static/overview.html:69]: I still don't know what CL looker is :-)

File: website/static/overview.html (hunk 1 above)
Comment (StevenMih) [reply website/static/overview.html:62]: hmmm, weird, this was supposed to be have all the commits but clearly its not working sorry - will follow up shortly.

**Comment by aweisberg** [comment]
Did you mean to close it? I see that everything is fixed except for my one question about what "Presto CLI Looker" is. Specifically not sure what looker is?

**Pull request closed without merging**